adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/model/Position.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.ticketmaster.model;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface Position {
+    int sectorId();
```

Comment:
@bearcheese 
My intention was to represent the id as simple as you would look up in a stadium.
"My position is located at sector: 24, seat: 43". 
While bookingId is more an identifier in the system like 10233556462396.